docs/installingNative.md

```diff
@@ -130,43 +119,48 @@ Now you can install the other dependencies.
   cd PrairieLearn
   ```
 
-!!! note "Setup a venv"
+- Set up a Python virtual environment:
 
-    It is recommended to use a virtual environment for Python dependencies. You can create a virtual environment within PrairieLearn:
-
-    === "Native"
+  === "uv"
```

Comment:
I made `uv` the first option here. I think it's currently the best way to install the `venv` and should be our recommended package management tool for python.